pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -3102,6 +3274,11 @@ do_file_io_operation(struct pt_regs *ctx, u32 event_id, u32 tail_call_id, bool i
         goto out;
     }
 
+    if (should_submit(TTY_WRITE, p.event)) {
```

Comment:
I am not sure if this is such a good idea, as this check is now in each `write` + `read` operation. Instead, we could hook [tty_write](https://elixir.bootlin.com/linux/v6.10.3/source/drivers/tty/tty_io.c#L1115). It should get called for `write` operations, as specified by the `file_operations` struct ([link to source](https://elixir.bootlin.com/linux/v5.4/source/drivers/tty/tty_io.c#L477)).